lib/mapper/product_mapper.rb

```diff
@@ -30,7 +30,7 @@ def self.map_payload(product, additional_data = {}, track_inventory = true, defa
                   width: product['width'] ? product['width']['value'] : '0',
                   inventory_tracking: isDigital || !track_inventory ? 'none' : 'product',
                 }
-                result[:upc] = product['gtin12'] if product['gtin12']
+                result[:upc] = product['isbn'] if product['isbn']
```

Comment:
@Jhalter5Stones This is not the correct change. Ignatius Press sells more than just books. Many products (CDs, DVDs, Art, etc) don't have an ISBN, and in such cases we _should_ be using the `gtin12`.

I think something like `isbn if it exists, gtin otherwise` is probably closer to what we want. (In order to achieve the ultimate UI changes, we _might_ also need to push a custom field of `upc_type` and use that as the _label_ in the UI.

Reply:
@mrogers-5s Ok I believe this should be fixed now. Let me know what you think